### PR TITLE
Allow annotation images from the file system

### PR DESCRIPTION
**Pull request by mattvot**
Adds support for using images in the file system as annotation images on both iOS and Android.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ annotations: [{
   title: 'Important',
   subtitle: 'Neat, this is a custom annotation image',
   annotationImage: {
-    source: { uri: 'https://cldup.com/7NLZklp8zS.png' },
+    source: { uri: 'file:///var/mobile/Containers/Data/Application/8821CE3F-C0DA-4B86-B4F7-85DE1E158943/Documents/custom-annotation.png' },
```
Comment (benvium) [inline API.md:314]: Perhaps we could say how you get this path in React Native, i.e. via `react-native-fs` or `react-native-fetch-blob`s File access APIs

Comment (mattvot) [reply API.md:314]: Sure, I can add this description under line 285

**Comment by henrikra** [comment]
@mattvot Wouldn't it be better if I could use local images like everywhere else like `require('path/to/my/image')`?

**Comment by benvium** [comment]
@henrikra That would be good too. This PR is useful for accessing marker images that have been downloaded to the file system (something that's handy for our use of this library here at Calvium).

**Comment by nitaliano** [comment]
Merging the `v6` PR in on Friday, sorry that we couldn't get this into `v5` we appreciate the PR.

**Pull request closed without merging**